src/agent/onefuzz-agent/src/worker.rs

```diff
@@ -212,11 +225,32 @@ impl State<Ready> {
 
         info!("IPC connection bootstrapped");
 
+        let log_path = Path::join(&self.ctx.work_dir, "task_log.txt");
```

Comment:
Don't forget to update this after your most recent commit

Reply:
No need, this this the path to the log file of the agent. 
My last commit updated the path to the log file in the container.